src/app/components/timeline/BeatTitleCell.js

```diff
@@ -229,6 +231,11 @@ class BeatTitleCell extends PureComponent {
         <Button bsSize={isSmall ? 'small' : undefined} block onClick={this.handleDelete}>
           <Glyphicon glyph="trash" />
         </Button>
+        {hierarchyLevel.level < 2 && (
```

Comment:
this is what i ended up doing on the horizontal hover options:
`const showExpandCollapse = hierarchyLevels.length - hierarchyLevel.level > 1`
and then in the return:
`showExpandCollapse && ( .....`